integration/util.go

```diff
@@ -14,7 +14,7 @@ import (
 )
 
 const (
-	defaultTimeout = 500
+	defaultTimeout = 600
```

Comment:
This is OK in the short term but we need to investigate if stack creation times are increasing by a large amount. Would be good to have an issue for that.

Reply:
Vol already created one. I added a comment. See https://github.com/giantswarm/giantswarm/issues/2717.